index.js

```diff
@@ -1 +1,114 @@
-// implement your API here
+const express = require('express')
+let users = require('./data/db.js')
+
+const app = express()
+ 
+app.use(express.json())
+
+app.get("/", (req, res) => {
+	res.send("Hey man");
+  });
+  
+  app.post("/api/users", (req, res) => {
+	const { name, bio } = req.body;
+  
+	if (!name || !bio) {
+	  res
+		.status(400)
+		.json({ error: "Please enter a name and a bio for the user" });
+	} else {
+	  users
+		.insert(req.body)
+		.then(user => {
+		  res.status(201).json(user);
+		})
+		.catch(error => {
+		  res
+			.status(500)
+			.json({ error: "There was an error processing your request" });
+		});
+	}
+  });
+  
+  app.get("/api/users", (req, res) => {
+	users
+	  .find()
+	  .then(users => {
+		res.status(200).json(users);
+	  })
+	  .catch(err => {
+		res.status(500).json({
+		  err,
+		  err: "the users information could not be retrieved"
```

Comment:
It's weird here.  Why do you have `{err, err: "the user..."}` . Based off this syntax the original key/value pair will be overwritten.